Assets/Scripts/CanonController.cs

```diff
@@ -0,0 +1,45 @@
+using UnityEngine;
+
+/// <summary>
+/// Controls the canon that shoots the boulders
+/// </summary>
+public class CanonController : MonoBehaviour
+{
+    public GameObject boulder;
+    private float _maxTimeBetweenShots;
+    private float _timeBetweenShots;
+    
+    [SerializeField] private float angle = 45f; // angle in DEGREES
```

Comment:
FYI your comment about this being in Degrees won't be seen by the IDE as documentation (unlike `Tooltip` or `///<summary/>`).
Maybe consider adding a tooltip like `[SerializeField, Tooltip("Angle (degrees)"), Range(-180,180)]`?